.changeset/dull-bags-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/select": patch
+---
+
+The PR makes changes to use the label placement in select according to the shouldLabelBeOutside rather than isOutsideLeft. This resolves the label placement issues in case of multiline.(fixes issue: #3841)
```

Comment:
- make it shorter
- ended with `(#3841)`

Reply:
Done in: 57bfd32